modules/odc-creating-apiserversource.adoc

```diff
@@ -31,6 +33,11 @@ image::apiserver-odc-config-1.png[Select the Service Account Name and Sink]
 +
 image::toplogy-odc-apiserver.png[ApiServerSource Topology view]
 
+[NOTE]
+====
+If a URI sink is used, the URI can be modified by right clicking on the *URI sink* -> *Edit URI*.
```

Comment:
If a URI sink is used, modify the URI by right-clicking on *URI sink* -> *Edit URI*.

Reply:
btw
` ```suggestion
blabla
``` `
ctrl+g or the icon :) github implemented magic stuff. Thanks for the review!